BaseClasses.py

```diff
@@ -755,19 +755,24 @@ def can_reach_region(self, spot: str, player: int) -> bool:
         return self.multiworld.get_region(spot, player).can_reach(self)
 
     def sweep_for_events(self, locations: Optional[Iterable[Location]] = None) -> None:
+        Utils.deprecate("sweep_for_events has been renamed to sweep_for_advancements. The functionality is the same. "
+                        "Please switch over to sweep_for_advancements.")
+        self.sweep_for_advancements(locations)
```

Comment:
```suggestion
        return self.sweep_for_advancements(locations)
```
Not needed, as it has no return, just "more correct".